src/features/goblins/tailor/TabContent.tsx

```diff
@@ -0,0 +1,188 @@
+  const [upcoming, setUpcoming] = useState<BumpkinShopItem[]>([]);
+  const [collection, setCollection] = useState<BumpkinShopItem[]>([]);
+  const [selected, setSelected] = useState<BumpkinShopItem | undefined>(
+    undefined
+  );
+
+  useEffect(() => {
+    const loadItems = async () => {
+      const upcoming = await loadCurrentAndUpcomingDrops();
+      const collection = await loadCollection();
+
+      setUpcoming(upcoming);
+      setCollection(collection);
+
+      const items = tab === "collection" ? collection : upcoming;
+
+      setSelected(items[0]);
+
+      setIsLoading(false);
+    };
+
+    loadItems();
+  }, []);
+
+  const items = tab === "collection" ? collection : upcoming;
+
+  const goToUpcomingDrops = () => {
+    window.open("https://testnet.bumpkins.io/#/upcoming-drops", "_blank");
+  };
+
+  const goToCollectionItem = () => {
+    window.open(
+      `https://testnet.bumpkins.io/#/collection/${selected?.tokenId}`,
+      "_blank"
+    );
+  };
+
+  if (isLoading) {
+    return (
+      <div className="h-60">
+        <span className="loading">Loading</span>
+      </div>
+    );
+  }
+
+  if (selected === undefined) {
+    return (
+      <div className="flex flex-col">
+        <span>Currently Unavailable!</span>
+        <span>Please try again later.</span>
+      </div>
+    );
+  }
+
+  const PanelDetail = () => {
+    if (tab === "collection") {
+      return (
+        <>
+          <span className="text-shadow text-center text-xs sm:text-sm leading-5">
+            {BUMPKIN_ITEMS[selected.name].description}
+          </span>
+          <div className="border-t border-white w-full mt-2 py-2 text-center">
+            <span className="text-shadow text-xxs text-center mt-2 sm:text-xs">
+              Visit Bumpkins.io for more info about this wearable
+            </span>
+          </div>
+          <Button className="text-xs mt-1" onClick={goToCollectionItem}>
+            Go to Bumpkins.io
+          </Button>
+        </>
+      );
+    }
+    const now = Date.now();
+    const releaseDate = selected.currentRelease?.releaseDate as number;
+    const releaseEndDate = selected.currentRelease?.endDate as number;
+    const mintIsLive = releaseDate < now && releaseEndDate > now;
+
+    return (
+      <>
+        <span className="text-shadow text-center text-xs leading-5">
+          {BUMPKIN_ITEMS[selected.name].description}
+        </span>
+        <div className="border-t border-white w-full mt-2 pt-2 text-center">
+          <span className="text-shadow text-center mt-2 text-xxs sm:text-xs">
+            {mintIsLive
+              ? "Minting is currently live on Bumpkins.io"
+              : "This wearable will be available to mint on Bumpkins.io"}
+          </span>
+          <div className="flex justify-center items-end my-1">
+            <img src={token} className="h-5 mr-1" />
+            <span className="text-xs sm:text-sm text-shadow text-center mt-2">
+              $25
```

Comment:
is this value constant for all wearables?

Reply:
🤦🏼‍♂️